src/datamodel_code_generator/reference.py

```
@@ -1190,6 +1196,26 @@ def get_valid_field_name_and_alias(
             field_name, excludes, class_name=class_name
         )
 
+    def resolve_default_value(
+        self,
+        field_name: str,
+        original_default: Any,
+        has_default: bool,  # noqa: FBT001
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `FBT001`)

Remove unused `noqa` directive

(RUF100)